openmoltools/utils.py

```diff
@@ -479,3 +485,54 @@ def get_checkmol_descriptors( molecule_filename, executable_name = 'checkmol' ):
  
     return descriptors
 
+def amber_to_gromacs( molecule_name, in_prmtop, in_crd, out_top = None, out_gro = None): 
+    """Use ParmEd to convert AMBER prmtop and crd files to GROMACS format.
+
+    Requires
+    --------
+    Currently requires gromacs_support branch of ParmEd
+
+
+    Parameters
+    ----------
+    molecule_name : str
+        String specifying name of molecule
+    in_prmtop : str
+        String specifying path to AMBER-format parameter/topology (parmtop) file
+    in_crd : str
+        String specifying path to AMBER-format coordinate file
+    out_top : str, optional, default = None
+        String specifying path to GROMACS-format topology file which will be written out. If none is provided, created based on molecule_name.
+    out_gro : str, optional, default = None
+        String specifying path to GROMACS-format coordinate (.gro) file which will be written out. If none is provided, created based on molecule_name.
+
+    Returns
+    -------
+    out_top : str
+        GROMACS topology file produced by ParmEd
+    out_gro : str
+        GROMACS coordinate file produced by ParmEd
+
+    Notes
+    -----
+        molecule_name is not currently used except to generate output file names if gro/top file names are not provided. It is an argument partly for API consistency.
```

Comment:
PS: I'm quite flexible if you find the `molecule_name` and `out_top` living as both inputs and ouputs confusing.  I think I've been sticking with this convention for legacy reasons, but I think streamlining your workflow is a compelling reason for cleaning up these legacy issues.  Obviously that would be a separate issue / pull request.